src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackNotes/FeedbackNotes.hooks.ts

```diff
@@ -104,18 +75,14 @@ export const useFeedbackNotes = ({ entity: { id, isFlow } }: FeedbackNotesProps)
     };
 
     if (isFlow && submitId) {
-      editFlowNote({ ...commonParams, submitId, flowId: id });
-
-      return;
+      editNote({ ...commonParams, submitId, flowId: id });
```

Comment:
let's move the condition logic `if (isFlow && submitId) {...}` as a part of `editNote` function